cmd/mdatagen/loader_test.go

```diff
@@ -36,17 +36,26 @@ func Test_loadMetadata(t *testing.T) {
 				ResourceAttributes: map[attributeName]attribute{
 					"string.resource.attr": {
 						Description: "Resource attribute with any string value.",
+						Enabled:     true,
 						Type: ValueType{
 							ValueType: pcommon.ValueTypeStr,
 						},
 					},
 					"string.enum.resource.attr": {
 						Description: "Resource attribute with a known set of string values.",
+						Enabled:     true,
 						Enum:        []string{"one", "two"},
 						Type: ValueType{
 							ValueType: pcommon.ValueTypeStr,
 						},
 					},
+					"resource.attr.explicitly.disabled": {
```

Comment:
Let's stick with the same default/optional terms as for metrics
```suggestion
					"optional.resource.attr": {
```